axolotl-web/src/pages/MessageList.vue

```diff
@@ -332,78 +330,56 @@ export default {
       }
     },
     scrollDown(){
-      if(document.getElementById("messageList-container")!=null)
-      document.getElementById("messageList-container").scrollTo(0,document.getElementById("messageList-container").scrollHeight);
+      var messageListContainer = document.getElementById("messageList-container");
+      if(messageListContainer != null) {
+        messageListContainer.scrollTo(0, messageListContainer.scrollHeight);
+      }
     },
     resetHeights(){
-      document.getElementById("messageInput").style.height="33px";
-    },
-    keyupHandler(e){
-      this.calcHeightsForInput(e);
+      document.getElementById("messageInput").style.height = "35px";
```

Comment:
There are way too many magic number in this, but at this PR is seriously broken at the moment, I'll let it like this for now.